dataplane/dataplane.cpp

```diff
@@ -265,6 +265,11 @@ eResult cDataPlane::init(const std::string& binaryPath,
 	return result;
 }
 
+eResult cDataPlane::DryRun(const std::string& configFilePath)
+{
```

Comment:
```suggestion
eResult cDataPlane::DryRun(std::string_view configFilePath)
{
```
We can use lightweight wrapper around `const char*` instead of creating dynamic `std::string`